Sources/StreamChat/APIClient/Endpoints/EndpointPath.swift

```diff
@@ -0,0 +1,87 @@
+//
+// Copyright © 2022 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+enum EndpointPath: Codable {
+    case connect
+    case sync
+    case users
+    case guest
+    case members
+    case search
+    case devices
+
+    case channels
+    case createChannel(String)
+    case updateChannel(String)
+    case deleteChannel(String)
+    case channelUpdate(String)
+    case muteChannel(Bool)
+    case showChannel(String, Bool)
+    case truncateChannel(String)
+    case markChannelRead(String)
+    case markAllChannelsRead
+    case channelEvent(String)
+    case stopWatchingChannel(String)
+    case pinnedMessages(String)
+    case uploadAttachment(String, String)
```

Comment:
Might be good to document the not so obvious associated values. For example, `uploadAttachment`, what do we pass in the two `String`s.